src/coin_test/backtest/trade_request.py

```diff
@@ -0,0 +1,37 @@
+"""Trade request object."""
+
+
+class TradeRequest:
+    """A request object for creating a trade."""
+
+    def __init__(
+        self,
+        symbol: str,
+        side: str,
+        type_: str = "market",
+        notional: float | None = None,
+        qty: float | None = None,
+    ) -> None:
+        """Initialize a new TradeRequest object.
+
+        Args:
+            symbol: The symbol of the asset being traded
+            side: buy or sell
+            type_: The type of trade, default is "market"
```

Comment:
Given this portion of the API will be public, this might be better as an enum?

Reply:
Done